src/SDKs/EdgeGateway/Microsoft.Azure.Management.EdgeGateway/Properties/AssemblyInfo.cs

```diff
@@ -7,7 +7,7 @@
 [assembly: AssemblyTitle("Microsoft Azure DataBox Edge Gateway Management Library")]
 [assembly: AssemblyDescription("Provides management functionality for Microsoft Azure DataBox Edge Gateway Management Resources.")]
 
-[assembly: AssemblyVersion("1.0.0.0")]
+[assembly: AssemblyVersion("0.9.0.0")]
```

Comment:
I probably misspoke earlier, we start versioning for new packages from `0.8.0-preview`
Please update the csproj and this file to set all versions to `0.8.0` (-preview in the csproj)